### PR TITLE
added todo-app files

### PR DESCRIPTION
**Pull request by rajanibusani**

### CODE REVIEW

**Review by Ekamba — CHANGES_REQUESTED** [review]
Hi @rajanibusani, instead of writing your function's name like this `Todoitems` you have to do this: `TodoItems`. In your RenderTodos I don't thing you have to wrap everything inside `<ol>...</ol>`. Remember olso to wrap li tag inside of <ul> like this `<ul> <li>..</li> </ul>`. You do not have wrap your component inside brace like you did inside App.js

File: react/week1/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Todolist from './todos';
+import './App.css';
+
+function App() {
+  return (
```
Comment (khjcph) [inline react/week1/todo-app/src/App.js:5]: maybe work a bit with the formatting here. for example

    return <div className="todo-app">{<Todolist />}</div>;

is a little easier to read

File: react/week1/todo-app/src/todos.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        deadline: "Fri Feb 26 2021"
+    },
+    {
+        id: 3,
+        task: "Book Dr appointment",
+        deadline: "Mon Mar 01 2021"
+    }
+]
+
+const Todoitems = ({ task, deadline }) => {
```
Comment (khjcph) [inline react/week1/todo-app/src/todos.js:19]: great that you destructure here!

File: react/week1/todo-app/src/todos.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+const RenderTodos = ({ todos }) => {
+
+    return (
+        <ol>
+            {
+                todos.map(todo => {
+                    return (
+                        <li  key={todo.id}>
+                            <Todoitems task={todo.task} deadline={todo.deadline} />
```
Comment (khjcph) [inline react/week1/todo-app/src/todos.js:35]: I don't know if this was covered in class, but lists in react should actually have a unique key. You can read more about those here: https://reactjs.org/docs/lists-and-keys.html#keys. If you would like to you can try to implement keys for your list.

File: react/week1/todo-app/src/todos.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+const Todoitems = ({ task, deadline }) => {
+    return (
+        <>
+            {task}, {deadline}
+        </>
+    )
+}
+
+const RenderTodos = ({ todos }) => {
```
Comment (khjcph) [inline react/week1/todo-app/src/todos.js:27]: great use of destructuring!

File: react/week1/todo-app/src/todos.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                            <Todoitems task={todo.task} deadline={todo.deadline} />
+                        </li>
+                    )
+                })}
+
+        </ol>
+    )
+}
+
+const Todolist = () => {
```
Comment (khjcph) [inline react/week1/todo-app/src/todos.js:44]: maybe this should be `TodoList`

**Review by khjcph — APPROVED** [review]
Great work!! 👍

I left a few comments that you can look at if you want. They are mostly about formatting. This is much easier if using a formatter / linter inside your code editor. I hope we will get to cover that in class :)

**Comment by rajanibusani** [comment]
> Great work!! 
> 
> I left a few comments that you can look at if you want. They are mostly about formatting. This is much easier if using a formatter / linter inside your code editor. I hope we will get to cover that in class :)

Thank you so much for your comments,i will change them as you suggested...tq

**Pull request merged**